server/routes/serviceProviderReferrals/editSessionView.ts

```diff
@@ -134,4 +140,104 @@ export default class EditSessionView {
+      },
+      errorMessage: ViewUtils.govukErrorMessage(this.presenter.fields.address.errors.addressLine1),
+      id: 'method-other-location-address-line-1',
+      name: 'method-other-location-address-line-1',
+      value: this.presenter.fields.address.value?.addressLine1,
+    }
+  }
+
+  private get otherLocationSecondAddressLineInputArgs(): InputArgs {
+    return {
+      label: {
+        html: 'Address line 2 (optional) <span class="govuk-visually-hidden">line 2 of 2</span>',
+      },
+      id: 'method-other-location-address-line-2',
+      name: 'method-other-location-address-line-2',
+      value: this.presenter.fields.address.value?.addressLine2,
+    }
+  }
+
+  private get otherLocationTownOrCityInputArgs(): InputArgs {
+    return {
+      label: {
+        text: 'Town or city',
+      },
+      errorMessage: ViewUtils.govukErrorMessage(this.presenter.fields.address.errors.townOrCity),
+      classes: 'govuk-!-width-two-thirds',
+      id: 'method-other-location-address-town-or-city',
+      name: 'method-other-location-address-town-or-city',
+      value: this.presenter.fields.address.value?.townOrCity,
+    }
+  }
+
+  private get otherLocationCountyInputArgs(): InputArgs {
+    return {
+      label: {
+        text: 'County',
+      },
+      errorMessage: ViewUtils.govukErrorMessage(this.presenter.fields.address.errors.county),
+      classes: 'govuk-!-width-two-thirds',
+      id: 'method-other-location-address-county',
+      name: 'method-other-location-address-county',
+      value: this.presenter.fields.address.value?.county,
+    }
+  }
+
+  private get otherLocationPostCodeInputArgs(): InputArgs {
+    return {
+      label: {
+        text: 'Postcode',
+      },
+      errorMessage: ViewUtils.govukErrorMessage(this.presenter.fields.address.errors.postcode),
+      classes: 'govuk-input--width-10',
+      id: 'method-other-location-address-postcode',
+      name: 'method-other-location-address-postcode',
+      value: this.presenter.fields.address.value?.postcode,
+    }
+  }
+
+  private meetingMethodRadioInputArgs(otherLocationHTML: string): RadiosArgs {
+    return {
+      idPrefix: 'meeting-method',
+      name: 'meeting-method',
+      fieldset: {
+        legend: {
+          text: 'Method',
+          isPageHeading: false,
+          classes: 'govuk-fieldset__legend--m',
+        },
+      },
+      errorMessage: ViewUtils.govukErrorMessage(this.presenter.fields.meetingMethod.errorMessage),
+      hint: {
+        text: 'Select one option.',
+      },
+      items: [
+        {
+          value: 'PHONE_CALL',
+          text: 'Phone call',
+          checked: this.presenter.fields.meetingMethod.value === 'PHONE_CALL',
+        },
+        {
+          value: 'VIDEO_CALL',
+          text: 'Video call',
+          checked: this.presenter.fields.meetingMethod.value === 'VIDEO_CALL',
+        },
+        {
+          value: 'IN_PERSON_MEETING_OTHER',
```

Comment:
i'm assuming we are holding off on allowing NPS office locations until we figure all that out?

Reply:
That is correct.